E2E/pageObjects/roleSelection.page.ts

```diff
@@ -18,4 +18,18 @@ export = {
     I.seeElement(locator.buttons.spectator);
     I.click(locator.buttons.voter);
   },
-};
\ No newline at end of file
+  
+  async loginByPressingEnter() {
```

Comment:
we have customerSteps.helper file. Clicking enter sounds like something that should be placed there.